sources/packages/backend/libs/integrations/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt.integration.service.ts

```diff
@@ -63,20 +63,13 @@ export class DisbursementReceiptIntegrationService extends SFTPIntegrationBase<D
   }
 
   /**
-   * Create file name of the daily disbursements records file.
+   * Create file name for the daily disbursements report file.
    * @param reportName Report name to be a part of filename.
-   * @returns Full file path of the file to be saved on the SFTP.
+   * @returns Full file path of the file to be sent via emails.
    */
-  createRequestFileName(reportName: string): {
-    fileName: string;
-    filePath: string;
-  } {
-    const timestamp = getFileNameAsCurrentTimestamp();
+  createDisbursementFileName(reportName: string): string {
+    const timestamp = getISODateOnlyString(new Date());
```

Comment:
Is there a requirement to have the "date only" added to the attachment name?
If we are looking for something to uniquely identify the file we may need to add the "File date" + "Sequence number".